include/internal/catch_commandline.cpp

```diff
@@ -170,6 +170,9 @@ namespace Catch {
             | Opt( [&]( bool flag ) { config.showDurations = flag ? ShowDurations::Always : ShowDurations::Never; }, "yes|no" )
                 ["-d"]["--durations"]
                 ( "show test durations" )
+            | Opt( config.minDuration, "seconds" )
+                ["-D"]["--min-duration"]
+                ( "show test durations when at least the given number of seconds" )
```

Comment:
```suggestion
                ( "show test durations for tests taking at least seconds" )
```
?

`when` definitely does not fit, but I am ok with changing it to just `with`.

Reply:
I went with your suggested change.